docs/explanations/faq.md

```diff
@@ -219,6 +219,11 @@ This is the canonical list of keyboard shortcuts:
 			<td><kbd>Esc</kbd></td>
 			<td><kbd>Esc</kbd></td>
 		</tr>
+		<tr>
+			<td>Select text between multiple blocks.</td>
```

Comment:
```suggestion
			<td>Select text across multiple blocks.</td>
```
I think you could argue we are not selecting text between blocks but across or spanning blocks

Reply:
💯